ordersettler/ordersettler.go

```diff
@@ -134,15 +135,9 @@ func (r *OrderSettler) findNewSettlements(ctx context.Context) error {
 				return fmt.Errorf("fetching current block height on chain %s: %w", sourceChainID, err)
 			}
 
-			// continue if settlement has already been ingested
-			if _, err := r.db.GetOrderSettlement(ctx, db.GetOrderSettlementParams{
-				SourceChainID:                     sourceChainID,
-				SourceChainGatewayContractAddress: sourceGatewayAddress,
-				OrderID:                           fill.OrderID,
-			}); err == nil {
```

Comment:
since we are not querying the db anymore, at startup won't we insert duplicate rows for all pending settlements that has yet to be settled on the source chain?

Reply:
ok i didn't really understand what this was doing I think i get this now ignore me